packages/zoe/src/zoeService/zoe.js

```diff
@@ -127,46 +110,93 @@ const makeZoeKit = (
     });
   };
 
-  /** @type {ZoeService} */
-  const zoeService = Far('zoeService', {
-    install: installBundle,
-    installBundleID,
-    startInstance,
-    offer,
-    /**
-     * @deprecated Useless but provided during transition to keep old
-     * code from breaking.
-     */
-    makeFeePurse: async () => feeMint.getFeeIssuer().makeEmptyPurse(),
-    /**
-     * @deprecated Useless but provided during transition to keep old
-     * code from breaking.
-     * @param {ERef<Purse>} _defaultFeePurse
-     */
-    bindDefaultFeePurse: _defaultFeePurse => zoeService,
-    getPublicFacet,
-
-    // The functions below are getters only and have no impact on
-    // state within Zoe
-    getInvitationIssuer: async () => invitationIssuer,
-    getFeeIssuer: async () => feeMint.getFeeIssuer(),
-    getBrands,
-    getIssuers,
-    getTerms,
-    getOfferFilter,
-    getInstallationForInstance,
-    getInstance,
-    getInstallation,
-    getInvitationDetails,
-    getConfiguration,
-    getBundleIDFromInstallation,
-    getProposalShapeForInvitation,
-  });
-
-  return harden({
-    zoeService,
-    feeMintAccess: feeMint.getFeeMintAccess(),
-  });
+  const makeZoeService = vivifyFarClassKit(
+    zoeBaggage,
+    'ZoeService',
+    ZoeServiceIKit,
+    dataAccess => ({ dataAccess }),
+    {
+      zoeService: {
+        install(bundleId) {
+          const { state } = this;
+          return state.dataAccess.installBundle(bundleId);
+        },
+        installBundleID(bundleId) {
+          const { state } = this;
+          return state.dataAccess.installBundleID(bundleId);
+        },
+        startInstance,
+        offer,
+        getPublicFacet(instance) {
+          const { state } = this;
+          return state.dataAccess.getPublicFacet(instance);
+        },
+
+        // The functions below are getters only and have no impact on
+        // state within Zoe
+        async getInvitationIssuer() {
```

Comment:
Is nice when the method list appears in the same order wherever the full list appears.